spec/registry.md

```diff
@@ -1,11 +1,34 @@
 # MessageFormat 2.0 Default Function Registry
 
-This section describes the functions for which each implementation MUST provide
-a _function handler_ to be conformant with this specification.
-
-Implementations MAY implement additional _functions_ or additional _options_.
-In particular, implementations are encouraged to provide feedback on proposed
-_options_ and their values.
+This section defines the **standard** _functions_ which are REQUIRED for conformance with this specification,
+along with **optional** _functions_ that SHOULD be implemented to support
+additional functionality.
+
+Implementations MUST define and support each **standard** _function_ and
+MUST accept all _options_ and _option_ values defined as **standard** listed in this specification.
```

Comment:
We ought to define somewhere what "accept" means.

Are there some conditions where an implementation could emit an error and use a fallback value for a placeholder which uses any set of options that it "MUST accept", or is it required that any perceived errors or unsupported values are silently ignored?

Reply:
To me, "accept" means that no error should be emitted just because a given function/option/option value appears in a placeholder. We say "accept" to avoid requiring that any given behavior attach to the f/o/v in question. However, it probably should work in some meaningful, recognizable manner.

Reply:
Done. See changes.